php/8-fpm/zzz-artifakt.ini

```diff
@@ -0,0 +1,24 @@
+memory_limit = 768M
```

Comment:
why is it 768M here ?
Maybe there should be a comment here explaining the reason behind this value

Reply:
value optimized for our PaaS, same value as in other PHP runtimes